cmd/create.go

```diff
@@ -0,0 +1,73 @@
+package cmd
+
+import (
+	"fmt"
+	"kool-dev/kool/cmd/builder"
+	"kool-dev/kool/cmd/presets"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+// KoolCreate holds handlers and functions to implement the preset command logic
+type KoolCreate struct {
+	DefaultKoolService
+	parser     presets.Parser
+	koolDocker builder.Command
```

Comment:
Instead of a command builder calling `kool docker`, you can use the `KoolDocker` service.